### ISSUE TITLE
How to divide training set and test set for DFAUST?

### ISSUE THREAD

**Issue by songtoy** [facebookresearch/VCMeshConv#11]
I'm curious about the model's ability to reconstruct different subjects. For example, training on subject A and test on subject B. Was the experiment mentioned in the paper implemented like this? Any suggestion would be appreciated.
